lib/helpers/isURLSameOrigin.js

```diff
@@ -28,7 +28,9 @@ module.exports = (
           href = urlParsingNode.href;
         }
 
-        isValidXss(url);
+        if (isValidXss(url)) {
```

Comment:
I think this should be moved up above line 25 since there is another `urlParsingNode.setAttribute` instance on the line 27.

Reply:
addressed in e76727886725c11a2dfa33472a8fe7d2647dfe07, thanks!